### ISSUE TITLE
error: java not available

### ISSUE THREAD

**Issue by dandanWang2019** [theaidenlab/3d-dna#167]
Hi hi,

I am running run-asm-pipeline.sh, and after 5 minutes, get error as below:
<img width="1083" alt="image" src="https://user-images.githubusercontent.com/55370145/226544159-783e522c-eb99-48d7-aa94-b7959f05ff29.png">

The software said java is not available, but I can use java directly on my linux system, so I do not know where should I add java path to. Do you have any idea about where is the problem?

Thanks a lot,
Dandan

**Comment by OririPla** [comment]
Hello,

I have exactly the same error, did you manage to fix it ?

Oriane

**Comment by dudcha** [comment]
Hello Oriane, Dandan,

The errors is caused by the following line:
type java >/dev/null 2>&1 || { echo >&2 ":( Java is not available, please install/add to path Java to run Juicer and Juicebox. Exiting!"; exit 1; }

You can try to run manually to try to debug.
Olga

**Comment by OririPla** [comment]
Hello Olga,

Thank you for answering so fast. I am not sure I understand what we are supposed to do. Delete the lines in the file ? I am sorry I am quite new to this.

Oriane

**Comment by dudcha** [comment]
No. Type the command in the terminal or submit the line as a separate job to understand why it is not running.

**Comment by OririPla** [comment]
Hello,

I managed to fix the error by specifying more clearly the java's paths in my bashrc (else in the cluster SLURM the job couldn't find java even if it existed).

Thank you for your time,

Oriane
